test/runtests.py

```diff
@@ -89,7 +89,11 @@ def test_path(folder, is_file):
                 print "Javascript file doesn't exist (" + js_file + ")"
                 sys.exit(1)
 
-            p = SP.Popen([ch_path, js_file], stdout=SP.PIPE, stderr=SP.STDOUT, close_fds=True)
+            working_path = os.path.dirname(js_file)
+            file_name = os.path.basename(js_file)
+            p = SP.Popen([ch_path, file_name], stdout=SP.PIPE, stderr=SP.STDOUT, 
+                                                             cwd=working_path,
+                                                             close_fds=True)
```

Comment:
I thought this X in mind. Good point on Linux. The initial plan was to make testing compatible to MT runs. i.e. Test multiple JS files at a time to reduce testing time. Still I can remove that and it should be fine.